packages/agents-hosting/src/app/auth/handlers/azureBotAuthorization.ts

```diff
@@ -393,6 +399,10 @@ export class AzureBotAuthorization implements AuthorizationHandler {
 
     if (!tokenResponse) {
       logger.debug(this.prefix('Cannot find token. Sending sign-in card'), activity)
+
+      if (!this._options.enableSso) {
+        delete (signInResource as any).tokenExchangeResource
+      }
       const oCard = CardFactory.oauthCard(this._options.name!, this._options.title!, this._options.text!, signInResource)
```

Comment:
Could not a boolean be added `CardFactory.oauthCard` to enable SSO or not?  As opposed to deleting that property?